documentation/modules/auxiliary/admin/kerberos/forge_ticket.md

```diff
@@ -213,3 +353,9 @@ Use the full hostname of the machine you are targeting, not just the domain:
 **Invalid SPN**
 
 SPNs must be in the format `*/*`. If this is not identical to what Active Directory is configured with, it will not work.
+
+**Verbose Mode**
```

Comment:
Just confirming, was this meant to be under the 'common mistakes' heading? 👀

Reply:
Hmmm I think I put it here thinking this is where people might look if something has gone wrong and here's how to debug it, but it doesn't quite fit in with the other items under the header, happy to move it under it's own header? `Debugging` maybe?